.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,369 @@
+            echo "sbt pact4s targets restored!"
+          else
+            echo "sbt pact4s targets need update!"
+          fi
+
+      - name: Verify consumer pacts and publish verification status to Pact Broker
+        id: verification-test
+        run: |
+          echo "provider-version=${{ env.PROVIDER_VERSION }}" >> $GITHUB_OUTPUT
+          echo "env.CONSUMER_NAME=${{ env.CONSUMER_NAME }}   # This reflects the consumer branch for pact verification (generated by Pact Broker)"
+          echo "env.CONSUMER_BRANCH=${{ env.CONSUMER_BRANCH }}   # This reflects the consumer branch for pact verification (generated by Pact Broker)"
+          echo "env.PROVIDER_BRANCH=${{ env.PROVIDER_BRANCH }}   # This reflects the provider branch to switch to for pact verification"
+          echo "env.CONSUMER_VERSION=${{ env.CONSUMER_VERSION }} # This reflects the consumer version for pact verification (generated by Pact Broker)"
+          echo "env.PROVIDER_VERSION=${{ env.PROVIDER_VERSION }} # Deprecate env.PACT_PROVIDER_COMMIT. This new envvar is used for migrating GIT hash to app versioning"
+          
+          # Refer to https://github.com/sbt/docker-sbt on this Docker image
+          # Recent sbt versions use Coursier instead of Ivy for dependency management.
+          # On Linux, the location is
+          #     sbt "show csrCacheDirectory"
+          # with a default value of ~/.cache/coursier/v1
+          docker run --rm -w /working \
+                          -v "$PWD":/working \
+                          -v ${{ env.COURSIER_CACHE }}:/root/.cache \
+                          -v ${{ env.SBT_CACHE }}:/root/.sbt \
+                          -e PACT_PUBLISH_RESULTS=true \
+                          -e PROVIDER_BRANCH=${{ env.PROVIDER_BRANCH }} \
+                          -e PROVIDER_VERSION=${{ env.PROVIDER_VERSION }} \
+                          -e CONSUMER_NAME=${{ env.CONSUMER_NAME }} \
+                          -e CONSUMER_BRANCH=${{ env.CONSUMER_BRANCH }} \
+                          -e CONSUMER_VERSION=${{ env.CONSUMER_VERSION }} \
+                          -e PACT_BROKER_URL=${{ env.PACT_BROKER_URL }} \
+                          -e PACT_BROKER_USERNAME=${{ env.PACT_BROKER_USERNAME }} \
+                          -e PACT_BROKER_PASSWORD=${{ env.PACT_BROKER_PASSWORD }} \
+                          -e POSTGRES_READ_URL=localhost:5432 \
+                          -e POSTGRES_USERNAME=rawls-test \
+                          -e POSTGRES_PASSWORD=rawls-test \
+                          sbtscala/scala-sbt:openjdk-17.0.2_1.7.2_2.13.10 \
+                          sbt -J-Xmx2g -J-XX:+UseG1GC \
+                          "project pact4s" clean coverage "testOnly org.broadinstitute.dsde.rawls.provider.*"  coverageReport
+
+  can-i-deploy: # The can-i-deploy job will run as a result of a Rawls PR. It reports the pact verification statuses on all deployed environments.
+    runs-on: ubuntu-latest
+    if: ${{ inputs.pb-event-type == '' }}
```

Comment:
How do the various `if` conditions for jobs interact?

`can-i-deploy` needs `verify-consumer-pact`, which needs `regulated-tag-job`, which only runs if `is-bump` is "no"?

Curious in what situation would we run `verify-consumer-pact` but not `can-i-deploy`?

Reply:
You can refer to [terra-billing-profile-manager](https://github.com/DataBiosphere/terra-billing-profile-manager/) to see how the formal tagging procedure works.

Reply:
I can see from the BPM Actions that this workflow is occasionally "Run manually by BroadBot". I'm guessing that's originating from Pact Broker.

> Curious in what situation would we run verify-consumer-pact but not can-i-deploy?

I still don't understand the distinction here. I thought the answer to "can I deploy?" is based on verifying consumer Pacts?